src/common/util.js

```diff
@@ -518,13 +517,17 @@ function printString(raw, options, isDirectiveLiteral) {
     rawContent,
     enclosingQuote,
     !(
-      options.parser === "css" ||
-      options.parser === "less" ||
-      options.parser === "scss"
+      isParser(options, "css") ||
+      isParser(options, "less") ||
+      isParser(options, "scss")
     )
   );
 }
 
+function isParser(options, name) {
```

Comment:
Can you rename this to `isLanguage`? It aligns with our plans to introduce that concept.

Reply:
Ideally the call would be `isLanguage(options, 'JSON')` so we wouldn't need to check `'json-stringify', `json5`, `json`. But it is ok for now.

Reply:
Or just check `astFormat`